Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -100,10 +100,11 @@ std::string GetDefaultSoundBackend()
 #elif defined __linux__
   if (AlsaSound::IsValid())
     backend = BACKEND_ALSA;
-  else
+  else if (CubebStream::IsValid())
     backend = BACKEND_CUBEB;
 #elif defined(__APPLE__) || defined(_WIN32) || defined(__OpenBSD__)
-  backend = BACKEND_CUBEB;
+  if (CubebStream::IsValid())
+    backend = BACKEND_CUBEB;
```

Comment:
Can we just move this out of the `#ifdef` soup and do it all the time? I'd expect that any platforms not in this chain (like `ANDROID` for example) to return `false` so it'd be skipped anyways.

Reply:
Yeah well, you could move it _in front_, but I guess either is fine then 🤷

Reply:
I think that would then affect Linux instead and make ALSA the default backend instead of Cubeb?

Reply:
Hm true. Bleh. Nevermind then I guess. But such define chains usually irk the heck out of me, and I try to avoid (or rewrite) them in favor of making it easier to follow. Not today I suppose.